src/pveclib/vec_char_ppc.h

```diff
@@ -212,6 +212,7 @@ vec_clzb (vui8_t vra)
   r = vec_vclzb (vra);
 #elif defined (__clang__)
   r = vec_cntlz (vra);
+#else
```

Comment:
Doesn't seem to match the rest of the patch. Is this tucking in a bug fix?

Reply:
Yes, found by inspection.